src/relation_engine_server/utils/arango_client.py

```diff
@@ -14,8 +14,10 @@
 
 def server_status():
     """Get the status of our connection and authorization to the ArangoDB server."""
+    auth = (_CONF['db_user'], _CONF['db_pass'])
+    adb_url = f"{_CONF['api_url']}/version"
     try:
-        resp = requests.get(_CONF['db_url'] + '/_api/database/current', auth=(_CONF['db_user'], _CONF['db_pass']))
+        resp = requests.get(adb_url, auth=auth)
```

Comment:
I don't really understand the difference between `/version` and `/_api/...`?

Reply:
The API in CI uses a user that only has access to the "ci" database, and any request to "/_api" gives a forbidden error. This request goes to "/_db/ci/_api/database/ 
/_api/version", which doesn't get a forbidden error.
Originally I was querying an admin cluster health endpoint. But that requires a full admin user.

Reply:
Do you mean `/_db/ci/_api/version`?
So `api_url` in conf is `/_db/ci/_api`?